vermouth/molecule.py

```diff
@@ -403,10 +403,6 @@ def force_field(self):
     def atoms(self):
         """
         All atoms in this molecule. Alias for `nodes`.
-
-        See Also
-        --------
-        :meth:`networkx.Graph.nodes`
```

Comment:
This is a lint?! Why not keep the the also?

Reply:
Not lint, no. But it breaks the doc building. See the commit description of efe3909. The root issue is that Graph.nodes is in the networkx intersphinx inventory as a property (as it should be), but there's no way to reference those in napoleon/autodoc/...